compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/AnnotationDesugar.java

```diff
@@ -766,7 +2132,7 @@ private BLangFunction defineFunction(Location pos, PackageID pkgID, BSymbol owne
         String funcName = ANNOT_FUNC + UNDERSCORE + annotFuncCount++;
         BLangFunction function = ASTBuilderUtil.createFunction(pos, funcName);
         function.setBType(new BInvokableType(Collections.emptyList(), symTable.mapType, null));
-
+        function.flagSet.add(Flag.PUBLIC);
```

Comment:
Why do we need this?

Reply:
to access annotations from another package.

Reply:
Why would we have to do that? Can you give an example?